environment.yml

```diff
@@ -29,6 +29,7 @@ dependencies:
   - types-protobuf
   - types-psycopg2
   - types-PyYAML
+  - transformers
```

Comment:
please also add the huggingface channel to the channels list?

Reply:
Ok, didn't know, thank you :) Now we install huggingface:transformers